services/slackServices/shoutout.js

```diff
@@ -75,7 +84,27 @@ exports.respondToInteractiveMessage = async reqInfo => {
 
       await slackModel.message.createDialog(dialog);
     } else if (reqInfo.actions[0].value === 'retrieve') {
-      console.log('retrieve');
+      const dialog = {
+        token: org.access_token,
+        trigger_id: reqInfo.triggerId,
+        dialog: JSON.stringify({
+          title: 'View shoutouts',
+          callback_id: 'view-shoutout',
+          submit_label: 'View',
+          elements: [
+            {
+              label: 'Who do you want to see their shoutouts?',
+              placeholder: 'Choose a person',
+              type: 'select',
+              name: 'user',
+              optional: false,
+              data_source: 'users',
+            },
+          ],
+        }),
+      };
+
+      await slackModel.message.createDialog(dialog);
     }
   } catch (err) {
     console.log(err);
```

Comment:
You may have forgotten to remove the `console.log(err);`

Reply:
I have changed this to return null. It's not a part of any HTTP request so I'm not what to do with it as the other slack services follow the same pattern. Let me know how I should proceed.